### PR TITLE
Dress Up 0.1.0.0: Inital Commit

### PR DESCRIPTION
**Pull request by Nifim**
Initial commit to port dress up

#198

### CODE REVIEW

File: dress_up/dress_up.lua [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            return group
+        end
+
+        error('Expected one of \'' .. table.concat(arg_lookups.groups, ',') .. '\', received \'' .. str .. '\'.')
+    end
+})
+
+-- Addon Command Handlers
+-- Five commands
+local du = command.new('du')
```
Comment (z16) [inline dress_up/dress_up.lua]: Here it says `/du` now, so I assume the `/do` up there was a typo?

File: dress_up/dress_up.lua [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+
+local blink = function(group, condition, bool)
+    if condition == 'all' then
+        for k in pairs(options.blink[group]) do
+          options.blink[group][k] = bool
+        end
+    else
+        options.blink[group][condition] = bool
+        settings.save()
```
Comment (z16) [inline dress_up/dress_up.lua]: You only save in the `else` case, was that intended?

File: dress_up/dress_up.lua [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE WINDOWER DEV TEAM BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+]]
```
Comment (z16) [inline dress_up/dress_up.lua]: New line :|

File: dress_up/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+**Date:** Nov. 23, 2019<br>
+
+# Dress Up #
+
+* Allows the swapping of character models. race, face, gender, and gear.
+* Prevents the blinking to keep targets selected.
+
+----
+
+**Main Command:** No full name command line. only `/do`
```
[the remark this answers is not in the document]
Comment (Nifim) [reply dress_up/README.md]: Typo, i meant `/du`. The "No full name command line" is just saying you cant do `/dress_up` i initially put that so people know you can only use `/du` but looking back it does seem unnecessary to say it explicitly.

File: dress_up/dress_up.lua (hunk 1 above)
Comment (Nifim) [reply dress_up/dress_up.lua]: correct

**Pull request merged**